### PR TITLE
v3

### PR DESCRIPTION
**Pull request by dominikzogg**

### CODE REVIEW

**Comment by dominikzogg** [comment]
@healerz please review the v3

File: src/Policy/CallbackPolicyIncludingPath.php [hunk 1]
```diff
@@ -5,7 +5,6 @@
 namespace Chubbyphp\Serialization\Policy;
 
 use Chubbyphp\Serialization\Normalizer\NormalizerContextInterface;
-use Chubbyphp\Serialization\SerializerLogicException;
 
 final class CallbackPolicyIncludingPath implements PolicyInterface
```
Comment (healerz) [inline src/Policy/CallbackPolicyIncludingPath.php]: Shall we rename "back" to `CallbackPolicy `in v3?

File: doc/Mapping/AdvancedNormalizationObjectMapping.md [hunk 2]
```diff
@@ -48,8 +48,8 @@ final class ModelMapping implements NormalizationObjectMappingInterface
     public function getNormalizationFieldMappings(string $path, string $type = null): array
     {
         return [
-            NormalizationFieldMappingBuilder::create('id')->setGroups(['baseInformation'])->getMapping(),
-            NormalizationFieldMappingBuilder::create('name')->setGroups(['baseInformation'])->getMapping(),
+            NormalizationFieldMappingBuilder::create('id')->getMapping(),
```
Comment (healerz) [inline doc/Mapping/AdvancedNormalizationObjectMapping.md]: How about we add the policy counterpart as an example here to illustrate the same behavior? `->setPolicy(new GroupPolicy('baseInformation'))`. I'd say having it in the advanced mapping example is enough, so not adding the policy example to all other examples where we removed the `setGroups` from the doc

File: src/Policy/CallbackPolicyIncludingPath.php [hunk 3]
```diff
@@ -19,18 +18,6 @@ public function __construct(callable $callback)
...
-     * @param mixed $object
-     *
-     * @throws SerializerLogicException
-     */
-    public function isCompliant(NormalizerContextInterface $context, $object): bool
-    {
-        throw SerializerLogicException::createDeprecatedMethod(__CLASS__, ['isCompliant']);
-    }
-
     public function isCompliantIncludingPath(string $path, object $object, NormalizerContextInterface $context): bool
```
Comment (healerz) [inline src/Policy/CallbackPolicyIncludingPath.php]: renaming it to `isCompliant`?

File: src/Policy/CallbackPolicyIncludingPath.php (hunk 1 above)
Comment (dominikzogg) [reply src/Policy/CallbackPolicyIncludingPath.php]: done

File: doc/Mapping/AdvancedNormalizationObjectMapping.md (hunk 2 above)
Comment (dominikzogg) [reply doc/Mapping/AdvancedNormalizationObjectMapping.md]: done

File: src/Policy/CallbackPolicyIncludingPath.php (hunk 3 above)
Comment (dominikzogg) [reply src/Policy/CallbackPolicyIncludingPath.php]: done

**Pull request merged**